### ISSUE TITLE
Can't use keyboard to rotate the cube on MacOS

### ISSUE THREAD

**Issue by yenchenlin** [davidwhogg/MagicCube#6]
Hello,
can it work on MacOS?

ping @jakevdp

**Comment by jakevdp** [comment]
Yes, but not on the OSX backend (for some reason I've not figured out). You can try it using one of the other available matplotlib backends, if you have one installed

**Comment by yenchenlin** [comment]
I see, thanks a lot!

**Comment by tpetrone** [comment]
Hi! Do you suggest any specific one?
